pipeline/tasks/protoc_tasks.py

```diff
@@ -455,3 +546,122 @@ def execute(self, gapic_api_yaml):
                 continue
             if 'package_name' in go_settings:
                 return go_settings.get('package_name')
+
+
+class PythonChangePackageTask(task_base.TaskBase):
```

Comment:
There is a lot of stuff here for Python - would it make sense to move it to a separate file?